app/main.py

```diff
@@ -1,8 +1,31 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for person_info in people:
+        name = person_info["name"]
+        age = person_info["age"]
+        person = Person(name, age)
+        person_list.append(person)
```

Comment:
This can be done in a few simpler 
```suggestion
    for  person_info in people:
        person_list.append(Person(person_info["name"], person_info["age"]))
```